picard/mbjson.py

```diff
@@ -380,24 +385,38 @@ def artist_credit_from_node(node):
         if 'joinphrase' in artist_info:
             artist_name += artist_info['joinphrase'] or ''
             artist_sort_name += artist_info['joinphrase'] or ''
-    return (artist_name, artist_sort_name, artist_names, artist_sort_names)
+    return (artist_name, artist_sort_name, artist_names, artist_sort_names, artist_countries)
+
+
+def _format_artist_countries(countries: list):
+    if not countries:
+        return ''
+    # Use an OrderedDict to get a list of unique countries in the original countries order.
+    unique_countries = OrderedDict()
+    for country in countries:
+        unique_countries[country] = None
+    return ', '.join(list(unique_countries.keys()))
```

Comment:
You can even simplify more using https://github.com/metabrainz/picard/blob/87b27f772d626addfa16f6217893285579330deb/picard/util/__init__.py#L590

Reply:
Nice!  Changed in [9e16ee0](https://github.com/metabrainz/picard/pull/2627/commits/9e16ee0394b1cee5d0939eed731f3de7b253bb1d).